x/dex/keeper/msg_server.go

```diff
@@ -103,12 +103,19 @@ func (k MsgServer) PlaceLimitOrder(
 	if err != nil {
 		return &types.MsgPlaceLimitOrderResponse{}, err
 	}
+	tickIndex := msg.TickIndexInToOut
+	if msg.PriceInToOut != nil {
+		tickIndex, err = types.CalcTickIndexFromPrice(*msg.PriceInToOut)
```

Comment:
Maybe it's better to allow using of only one field? Either `.TickIndexInToOut` or `.PriceInToOut`?

Reply:
This is enforced in validate basic. Since the plan is to get rid of TickIndexInToOut it doesn't make sense to use a proto ONEOF field